### ISSUE TITLE
Bug: ElectrostaticsHandler's scale14 is not wired up correctly

### ISSUE THREAD

**Issue by j-wags** [openforcefield/openff-toolkit#408]
**Describe the bug**
It's not clear that the [`ElectrostaticsHandler`'s `scale14` attribute](https://github.com/openforcefield/openforcefield/blob/master/openforcefield/typing/engines/smirnoff/parameters.py#L2409) plugs into anything. We should make sure that it reaches the System's `Exceptions` list. The code will probably look similar to that of [`vdWHandler`](https://github.com/openforcefield/openforcefield/blob/master/openforcefield/typing/engines/smirnoff/parameters.py#L2392-L2393)

**Comment by mattwthompson** [comment]
So this stems from how vdW and electrostatic 1-4 scaling is treated differentely in OFFTK but handled at the [same time](https://github.com/openmm/openmm/blob/ef02e48dbc09f2a6f8e8ce2b22010887d2585fe2/wrappers/python/simtk/openmm/app/forcefield.py#L2415) in OpenMM. It [looks like](https://github.com/openforcefield/openforcefield/blob/master/openforcefield/typing/engines/smirnoff/parameters.py#L2530-L2531) it's just falling back to 5/6 - is this something that is tested, i.e. that the value in the XML is actually translated into the OpenMM System?

If we can assume that `vdWHandler. postprocess_system` is _always called before_ `ElectrostaticsHandeler.postprocess_system`, we could set the electrostatic 1-4 scaling factor to 1 initially, while the LJ parameters are being set, and clean them up later (`this_exception.somehow_get_electrostatics14 *= self._scale14`). But this is obviously a bad idea.

The way I see it, the options are 
1. Have each of these two handlers cover the cases of
  * The exceptions don't already exist, so create them based on the known scale14 value for this handler and assuming 1 (or some other value) as a placeholder
  *  The exceptions already exist but need to be updated to reflect the relevant scale14 value
  * (optional/ugly) iterate through a second time and make sure they are valid
2. Only have these handlers do the things they directly know about, and do some cleaning up to handle the case of handlers crossing over each other, i.e. [here](https://github.com/openforcefield/openforcefield/blob/af7c2f862c42ab486a3bf1ec90d36920862faba3/openforcefield/typing/engines/smirnoff/forcefield.py#L1140-L1144) before the return. This isn't pretty but it maintains most of the independence between the different handlers.
3. Refactor these two handlers to a single "super non-bonded handler" that stores all non-bonded interactions

**Comment by j-wags** [comment]
Bumping this -- We just ran into it again during some work on OpenFF System -- This is in the "silent error that gives different numbers" category, so we should really get this fixed now.

We could make a test that makes two systems, using FFs that vary by only by their 1-4 scaling factors, and manually checks some exceptions in the output OpenMM systems to ensure they're properly applied. 

We should also look for all uses of `0.83...` in the OFF Toolkit to ensure that they're all the same precision. These numbers are different in the OFFXML files, since at some point we switched form recording it in ~5 digits to ~10.
